Releases.md

```diff
@@ -1,3 +1,13 @@
+### 0.138.0 / 2022.05.05
+
+- :bug: Export equalSimd and equalsNaive (#2187)
```

Comment:
Ooops, I made a mistake, can you change it to `fix:` prefix?

Reply:
Sure. Done.